source/partials/release_notes/_release-17-5-0.md.erb

```diff
@@ -73,7 +73,7 @@ Features that become superseded or irrelevant become *deprecated*, and are sched
 
 The following features have been deprecated. Please note the timeframe of when they will be removed.
 
-* Support for in-built password file and LDAP/AD authentication have been deprecated as of this release. Support for these authentication mechanisms would be removed in 17.9 release (scheduled to be released in September 2017).
+* Support for in-built password file and LDAP/AD authentication have been deprecated as of this release. Support for these authentication mechanisms would be removed in 17.10 release (scheduled to be released in October 2017).
```

Comment:
Is it ok to modify the previous release notes? I know we might remove support for these plugins one release late, but I think it should still be ok over changing the previous release notes.